plugins/src/test/groovy/co/bound/codeartifact/CodeArtifactPluginTest.groovy

```diff
@@ -89,6 +96,52 @@ class CodeArtifactPluginTest extends PluginTest {
         result.output.contains("Searched in the following repositories:")
         result.output.contains("codeartifact(${wiremock.baseUrl()}/${domain}-${accountId}.d.codeartifact.${region}.amazonaws.com/maven/$repo/)")
         wiremock.verify(1, newRequestPattern(POST, urlMatching("/v1/authorization-token.*")))
+        wiremock.verify(1, newRequestPattern(GET, anyUrl()))
+
+        where:
+        gradleVersion << gradleVersions()
+    }
+
+    def "can configure repository filters"() {
+        given:
+        givenCodeArtifactWillReturnAuthToken()
+        def settingsKt = file('settings.gradle.kts')
+        settingsFile.renameTo(settingsKt)
+        settingsFile = settingsKt
+        settingsFile.setText("""
+            plugins {
+                id("co.bound.codeartifact")
+            }
+            codeartifact {
+                domain.set("$domain")
+                accountId.set("$accountId")
+                region.set("$region")
+                repo.set("$repo")
+                val contentAction = {
```

Comment:
For some reason inlining the lambda does not work, as kotlin gets confused what `Property.set` function to call.